activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add additional database-specific rake tasks for multi-database users
```

Comment:
Can change this to mention the specific tasks you added? This is currently a little vague since we already had create/drop/migrate etc

Reply:
Yep, of course

Reply:
I've expanded on this section. It now mentions what we already had (create/drop/migrate) and what we are adding (schema:dump/schema:load/structure:dump/structure:load/test:prepare)